client/my-sites/plugins/plugin-management-v2/plugin-action-status/style.scss

```diff
@@ -26,7 +26,8 @@
 	}
 }
 
-.plugin-action-status-inProgress {
+.plugin-action-status-inProgress,
+.plugin-action-status-upToDate {
```

Comment:
I don't think we use uppercase on class names. I recommend using `-inprogress` and `-uptodate` to follow the conventions.

Reply:
I have changed `upToDate` to `up-to-date`. `inProgress` status is used in quite a lot of places, so leaving it as it is for now.